forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -200,10 +200,6 @@ const Edit = React.memo((props) => {
                 data["processName"]= processListData.processName;
               }
               
-              // keep the existing status
-              if (processListData.status) {
-                data["status"]= processListData.status;
-              }
```

Comment:
Turned out this was not necessary. Removing to keep the code cleaner.

Reply:
Noted! thanks for fixing this bug Iman :)